src/renderer/component/publishForm/view.jsx

```diff
@@ -360,10 +363,16 @@ class PublishForm extends React.PureComponent<Props> {
           <section className="card card--section">
             <div className="card__title">{__('Thumbnail')}</div>
             <div className="card__subtitle">
-              {__(
-                'Upload your thumbnail to spee.ch, or enter the url manually. Learn more about spee.ch '
+              {uploadThumbnailStatus === STATUSES.API_DOWN ? (
```

Comment:
How feasible (and beneficial) would it be to split the entirety of this into a separate component?

Reply:
Not hard. We could just move this into the `SelectThumbnail` component.

Reply:
Until we move add this functionality somewhere else I think it's ok right now